src/languages/lambda/code.js

```diff
@@ -0,0 +1,349 @@
+"use strict";
```

Comment:
`code.js` isn't a great name for this file as it doesn't really describe what the file is about. Perhaps call it `parser.js` instead?